src/main/java/org/linksharing/server/http/ApiRestController.java

```diff
@@ -69,18 +73,38 @@ ResponseEntity<UserProfileDetails> updateProfileDetails(Principal user,
     }
 
     @GetMapping("/profile/picture")
-    ResponseEntity<?> getProfilePicture(Principal user) throws FileNotFoundException {
+    ResponseEntity<?> getProfilePicture(Principal user) throws IOException {
 
-        File img = new File("../image.jpg");
+        UserProfileDetails userProfile = profileRepository.findByEmail(user.getName());
+        String imageUrl;
+
+        if (userProfile.getImageUrl() != null) {
+            imageUrl = userProfile.getImageUrl();
+        } else {
+            imageUrl = "src/main/resources/static/placeholder.jpg";
```

Comment:
https://github.com/eugenesimakin/link-sharing-app/pull/14#discussion_r1436044526

Reply:
Oh, crap)
This path should be relative also. I got that.